### DISCUSSION TITLE
Open issues for 0.16.10 release

### DISCUSSION THREAD

**Discussion by primo-ppcg** [micjahn/ZXing.Net#514] [Q&A] [answered]
I would like to ask if there are any open issues for the 0.16.10 release, so that I might help resolve some of them. The [milestones list](https://github.com/micjahn/ZXing.Net/milestones) doesn't appear to have been updated since 0.16.6.

**Comment by micjahn** [comment, marked as answer]
Sorry, I haven't used the milestones since 0.16.6. There are a lot of open issues on the list. But they are not planned for 0.16.10. I'm actually working on the next release. But I want to port the last changes of the java version first.
